demo/apiExamples.md

```diff
@@ -76,7 +76,48 @@ A baseline `<auro-select>` using `<auro-menu>` and `<auro-menuoption>` elements.
 
 ### Property Examples
 
-#### <a name="placeholder"></a>`placeholder`
+#### value
+
+Use the `value` property to define a preset value on the `auro-select` element. The `value` of `auro-select` must match the `value` defined of a single `auro-menuoption`. Upon initially rendering the component, if the `value` does not match an `auro-menuoption`, the `value` of `auro-select` will be set to `undefined`.
+
+While `auro-menu` supports presetting the selection of an `auro-menuoption` by applying the `selected` attribute on the `auro-menuoption`, that should never be done when directly setting the `value` property on `auro-select`.
```

Comment:
Why can we not have the pre-selected value also be supported by the `selected` attribute?